content/doc/book/blueocean/activity.adoc

```diff
@@ -18,32 +18,69 @@ repository Pipeline.
 
 image:blueocean/activity/overview.png[Overview of the Activity View, role=center]
 
-
 == Activity
 
+The default subsection of Activity Viewer shows a list of the latest
+completed or in-progress Runs of a Pipeline.
```

Comment:
@rtyler - Okay, well, it is easy enough to fix over capitalization in the final edit.